shared/src/business/entities/contacts/Contact.test.ts

```diff
@@ -2,13 +2,13 @@ import { Contact } from './Contact';
 import { MOCK_CONTACT_PRIMARY } from '../../../test/mockContact';
 
 describe('Contact', () => {
-  describe('hasEAccess validation', () => {
+  describe('hasElectronicAccess validation', () => {
```

Comment:
We renamed columns with `E` abbreviation for two reasons:
1. It was not consistent with our other, non-abbreviated nomenclature
2. Our library had problems converting to/from Postgres snake_case